djangocms_alias/views.py

```diff
@@ -93,13 +94,20 @@ class AliasListView(PermissionRequiredMixin, ListView):
     template_name = 'djangocms_alias/alias_list.html'
 
     def get_queryset(self):
+        site = self.request.GET.get('site', None)
```

Comment:
We have to sanitise this input, this will create a security issue, especially as the value is immediately used to create a query.